vnext/ReactWindowsCore/WinRTWebSocketResource.cpp

```diff
@@ -237,15 +273,36 @@ fire_and_forget WinRTWebSocketResource::PerformWrite()
       self->m_writer.WriteBytes(arr);
     }
 
-    co_await self->m_writer.StoreAsync();
+    auto async = self->m_writer.StoreAsync();
 
-    if (self->m_writeHandler)
+    co_await lessthrow_await_adapter<DataWriterStoreOperation>{async};
+
+    if (SUCCEEDED(async.ErrorCode()))
```

Comment:
S_OK ?

Reply:
S_OK is part of Wndows.h. I think C++/WinRT uses 0.

---
In reply to: [440554865](https://github.com/microsoft/react-native-windows/pull/5174#discussion_r440554865) [](ancestors = 440554865)